### PR TITLE
fix responses

### PR DESCRIPTION
**Pull request by Sanchit011**

### CODE REVIEW

**Comment by service-0chain** [comment]
Manual system tests [failure] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | ABILITY TO SKIP TESTS IS DISABLED. SKIP_TESTS OPTION IGNORED.  |
| system-tests  | fix/update-delete  |
| gosdk  | NOT OVERRIDDEN  |
| zboxcli  | staging  |
| zwalletcli  | staging  |
| 0chain  | staging  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | feat/delete-routes  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | master  |
| 0proxy  | staging  |
| 0search  | staging  |

**Comment by service-0chain** [comment]
Manual system tests [failure] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | fix/update-delete  |
| gosdk  | NOT OVERRIDDEN  |
| zboxcli  | staging  |
| zwalletcli  | staging  |
| 0chain  | staging  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | feat/delete-routes  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | master  |
| 0proxy  | staging  |
| 0search  | staging  |

**Comment by service-0chain** [comment]
Manual system tests [cancelled] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | fix/update-delete  |
| gosdk  | NOT OVERRIDDEN  |
| zboxcli  | staging  |
| zwalletcli  | staging  |
| 0chain  | staging  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | feat/delete-routes  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | master  |
| 0proxy  | staging  |
| 0search  | staging  |

File: tests/api_tests/0box_test.go [hunk 1]
```diff
@@ -563,7 +563,7 @@ func Test0boxNft(testSetup *testing.T) {
 		)
 		require.NoError(t, err)
 		require.NotNil(t, zboxNftUpdated)
-		require.Equal(t, 201, response.StatusCode(), "Response status code does not match expected. Output: [%v]", response.String())
+		require.Equal(t, 200, response.StatusCode(), "Response status code does not match expected. Output: [%v]", response.String())
```
Comment (stewartie4) [inline tests/api_tests/0box_test.go:566]: HTTP status code 201 means "created"
This is inappropriate for PUT/PATCH calls which updates an already existing resource

File: tests/api_tests/0box_test.go [hunk 2]
```diff
@@ -609,7 +609,7 @@ func Test0boxNft(testSetup *testing.T) {
 		// require.Error(t, err) Fixme error is not send in error instead it is send
 		require.NoError(t, err)
 		require.NotNil(t, zboxNftCollectionId)
-		require.Equal(t, 201, response.StatusCode(), "Response status code does not match expected. Output: [%v]", response.String())
+		require.Equal(t, 200, response.StatusCode(), "Response status code does not match expected. Output: [%v]", response.String())
```
Comment (stewartie4) [inline tests/api_tests/0box_test.go:612]: HTTP status code 201 means "created"
This is inappropriate for PUT/PATCH calls which updates an already existing resource

**Review by stewartie4 — CHANGES_REQUESTED** [review]
Some API changes seem incorrect

File: tests/api_tests/0box_test.go (hunk 1 above)
Comment (Sanchit011) [reply tests/api_tests/0box_test.go:566]: we should use 200 right?
that is what the new status code is

**Comment by service-0chain** [comment]
Manual system tests [success] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | fix/update-delete  |
| gosdk  | NOT OVERRIDDEN  |
| zboxcli  | staging  |
| zwalletcli  | staging  |
| 0chain  | staging  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | feat/delete-routes  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | master  |
| 0proxy  | staging  |
| 0search  | staging  |

**Review by Kishan-Dhakan — APPROVED** [review]
LGTM

**Pull request merged**